truss/tests/contexts/image_builder/test_serving_image_builder.py

```diff
@@ -11,7 +11,7 @@
 
 def test_serving_image_dockerfile_from_user_base_image(custom_model_truss_dir):
     th = TrussHandle(custom_model_truss_dir)
-    th.set_base_image("baseten/truss-server-base:3.9-v0.4.3")
+    th.set_base_image("baseten/truss-server-base:3.9-v0.4.3", "usr/local/bin/python3")
```

Comment:
Curios why this doesn't start with a slash, i.e. why it isn't `/usr/local/bin/python3`.

Reply:
good catch, this should be the full path – this test doesn't make use of the executable so it was missed